.github/workflows/name.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] [OPTIONAL] kindly delete this file `(name.txt)` since its not useful and empty

Reply:
The file has been removed. Thank you